### PR TITLE
Add deleteD command

### LINKED ISSUE #97: Add deleteD command

### PR DESCRIPTION
**Pull request by vevek**
Closes #97

### CODE REVIEW

**Comment by vevek** [comment]
Ready for review. Will add tests soon after merge.

File: src/main/java/seedu/address/logic/commands/DeleteDeadlineCommand.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            + "Parameters: PROJECT_INDEX (must be a positive integer)"
+            + PREFIX_REMOVE_TASK_INDEX + "DEADLINE_INDEX \n"
+            + "Example: " + COMMAND_WORD + " 1" + " "
+            + PREFIX_REMOVE_TASK_INDEX + " 2";
+
+    private final Index projectIndex;
+    private final Index targetDeadlineIndex;
+
+    /**
+     * Creates a DeleteDeadlineCommand to delete the specified {@code Deadline} from {@code Project}.
```
Comment (samuelfangjw) [inline src/main/java/seedu/address/logic/commands/DeleteDeadlineCommand.java:32]: blank line

File: src/main/java/seedu/address/logic/parser/DeleteDeadlineCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new DeleteDeadlineCommand object
+ */
+public class DeleteDeadlineCommandParser implements Parser<DeleteDeadlineCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteDeadlineCommand
+     * and returns a DeleteDeadlineCommand object for execution.
```
Comment (samuelfangjw) [inline src/main/java/seedu/address/logic/parser/DeleteDeadlineCommandParser.java:20]: blank line

**Review by samuelfangjw — APPROVED** [review]
LGTM

**Pull request merged**